verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -24,6 +24,29 @@
 namespace verilog {
 namespace kythe {
 
+void KytheFactsExtractor::ExtractKytheFacts(const IndexingFactNode& root) {
+  CreatePackageScopes(root);
+  Visit(root);
+}
+
+void KytheFactsExtractor::CreatePackageScopes(const IndexingFactNode& root) {
+  for (const verible::VectorTree<IndexingNodeData>& child : root.Children()) {
+    if (child.Value().GetIndexingFactType() != IndexingFactType::kPackage) {
+      continue;
+    }
+
+    Visit(child);
```

Comment:
How do you prevent outputting duplicate VNames? (Visit is called for every package child, and for the code root)

Reply:
Thanks for this note, I didn't notice that.